fossas/fossa-cli
# [ANE-283] Adds sbt dependency tree strategy

# Overview

This PR adds a strategy for `sbt dependencyTree` for single project builds.

## Acceptance criteria

- As a user, when I analyze sbt project (single project build), `sbt dependencyTree` tactic is attempted

## Testing plan

1. Run `make install-dev`
2. Run `fossa-dev analyze -o | rendergraph` against any of the following candidates:

Candidates:
- https://github.com/fossas/scala3-example-project
- https://github.com/fossas/example-projects
- https://github.com/pauljamescleary/scala-pet-store

* Note that deep dependencies are included
* Note that graph matches that of `sbt depedndencyTree` (note: `rendergraph` and `sbt dependencyTree` may render dependencies in a different order! Also `rendergraph` includes unused (dev/test) dependency in it's output)

## Risks

N/A

## References

https://fossa.atlassian.net/jira/software/c/projects/ANE/boards/39?modal=detail&selectedIssue=ANE-283

## Checklist
